css/css-layout-api/constraints-data-sab-failure.https.html

```diff
@@ -35,14 +35,9 @@
     let childFragment = null;
 
     try {
-      // We need SABs to be enabled to properly run this test.
-      if (typeof SharedArrayBuffer !== 'undefined') {
-        childFragment = yield child.layoutNextFragment({
-          data: { sab: new SharedArrayBuffer(4) }
-        });
-      } else {
-        throw Error();
-      }
+      childFragment = yield child.layoutNextFragment({
+        data: { sab: new SharedArrayBuffer(4) }
+      });
     } catch(e) {
```

Comment:
This also seems kinda wrong as the specific error is not checked. Why is this not using assert_throws?

Reply:
Currently the test passes if SAB isn't supported (but the API otherwise works). I assume that's intentional. If so, then it can't just use assert_throws, as that method can only check for one type of error at a time.

But I agree that restructuring it to use assert_throws (rather than a catch and then implicit success) would be better.

Reply:
SAB shouldn't be optional.

Reply:
I'm sorry, your response is too terse for me to understand. Are you saying that it's okay for the test to fail if SAB isn't supported? This isn't *intended* to be a test for SAB support.

Reply:
The test is only relevant if SharedArrayBuffer is supported. Having a test always succeed when a feature it's testing isn't supported is a not a pattern we generally follow and I'd rather we don't, ever.

Reply:
That's reasonable.